lib/rubocop/cop/style/mixin_usage.rb

```diff
@@ -48,24 +48,22 @@ class MixinUsage < Cop
 
         def_node_matcher :include_statement, <<-PATTERN
           (send nil? ${:include :extend :prepend}
-            (const nil? _))
+            const)
         PATTERN
 
         def on_send(node)
           return unless (statement = include_statement(node))
```

Comment:
We will normally use a block here, i.e.:

```ruby
include_statement(node) do |statement|
  return unless implicit_style?(node)

  add_offense(node, message: format(MSG, statement: statement))
end
```

Ideally the node matcher would integrate the call to `#implicit_style?` through a funcall as well. 🙂